src/foam/u2/view/RichChoiceView.js

```diff
@@ -596,7 +596,18 @@ foam.CLASS({
                   }));
               }))
           } else {
-            return self.E().add(fullObject_ ? fullObject_.toSummary() : '');
+            return self.E()
+              .addClass(this.myClass())
+              .start()  
```

Comment:
Do we need this `.start().end()` wrapper?

Reply:
yup its needed to position the selectionview correctly